OpenDreamShared/Compiler/DMPreprocessor/DMPreprocessorLexer.cs

```diff
@@ -196,10 +212,10 @@ class DMPreprocessorLexer : TextLexer {
                     base.Advance();
 
                     current = Advance();
-                    while (current == ' ' || current == '\t' || current == '\n')
-                    {
-                        current = Advance();
-                    }
+//                    while (current == ' ' || current == '\t' || current == '\n')
+//                    {
+//                        current = Advance();
+//                    }
```

Comment:
Why was this commented out? When combining lines, the whitespace at the beginning of the next line is removed.

Reply:
Reverted, commenting it out does not seem to cause errors, but stripping the newlines here is probably causing errors in expanded macros that have line breaks at the end of their final closing brace. When there are two #defines separated by a 'newline only' line the preproc errors because the second #define ends up on the line where the previous macro is ending.